src/katgpucbf/xbgpu/correlation.py

```diff
@@ -292,7 +292,7 @@ def reduce(self) -> None:
 
     def zero_visibilities(self) -> None:
         """Zero all the values in the internal buffer."""
-        self.ensure_all_bound()
```

Comment:
Why did we do `ensure_all_bound` in the first place? Was it just easier? Slash, we didn't really envision being in this situation?

Reply:
I think `ensure_bound` was only added to katsdpsigproc later, to address this sort of situation.